types/three/three-core.d.ts

```diff
@@ -5389,17 +5389,72 @@ export class Skeleton {
     clone(): this;
 }
 
+/**
+ * A mesh that has a Skeleton with bones that can then be used to animate the vertices of the geometry. The material must support skinning and have skinning enabled.
+ * @see MeshStandardMaterial.skinning
+ * @source https://github.com/mrdoob/three.js/blob/master/src/objects/SkinnedMesh.js
+ */
 export class SkinnedMesh extends Mesh {
-    constructor(geometry?: Geometry | BufferGeometry, material?: Material | Material[], useVertexTexture?: boolean);
 
+    /**
+     * @param  {BufferGeometry} geometry? an instance of {@link BufferGeometry}.
+     * @param  {Material|Material[]} material? (optional) an instance of {@link Material}. Default is a new {@link MeshBasicMaterial}.
+     * @param  {boolean} useVertexTexture?
+     */
+    constructor(geometry?: Geometry | BufferGeometry, material?: Material | Material[]);
```

Comment:
the type of `geometry` doesn't match the PR description; it's still `Geometry | BufferGeometry`.

Reply:
Please check my comment above:
[https://github.com/DefinitelyTyped/DefinitelyTyped/pull/33308#issuecomment-466497645](url)
I had to keep "Geometry | BufferGeometry" in order to pass the obsolete Travis CI Tests, so this PR is not rejected and you could review it (nobody replied to me with an answer, though).